### PR TITLE
new clones (teedoff)

### PR DESCRIPTION
**Pull request by DavidHaywood**
new clones
---------
Tee'd Off (World) [system11]

note, since the new set has the same 'protection' code that we thought might be due to a bad dump, I've unflagged the other ROMs.

this is the new parent set.  I've also demoted the game to NOT WORKING for now because there's something strange going on, and it isn't clear that the behavior is 100% correct with the patch.

system11 also provided the following PCB images http://mamedev.emulab.it/haze/amisc/teedoff_pcb_images.zip note @angelosa there doesn't seem to be anything out of the ordinary anywhere on the PCB.  Maybe this code shouldn't be being executed at all and the failure is earlier? maybe it's an open bus read?

### CODE REVIEW

**Comment by angelosa** [comment]
I think that the correct flag here is `MACHINE_UNEMULATED_PROTECTION`, without MNW since afaik there's no clear protection mechanism other than the attract mode not taking wind into account.
At this point I'm pretty positive that whatever is enabled by master CPU at 0xf850 should be a view window of some sort, that overlays shared RAM with the necessary code snippet. If it's not the M5205 layer (that I haven't checked) then guessing this is near to impossible without schematics.

**Comment by DavidHaywood** [comment]
there's no protection device, it might not be protection.. it might just be some weird debugging leftover if it's some kind of open bus check.

**Comment by angelosa** [comment]
imho if it does some non-canonical uploading by using a Z80 sorcery that counts as protection, but I defer to @cuavas  for the final decision on this.

**Comment by ajrhacker** [comment]
The only remotely suspicious Z80-type thing going here is both the main and sub CPUs setting and clearing bits in the I register, which the Z80 automatically outputs on A15-A8 during refresh cycles as well as IM 2 vectoring cycles. The latter is irrelevant here because IM 1 is set for both CPUs.

I'm also inclined to believe that the palette RAM might just be write-only on this board.

File: src/mame/tecmo/tehkanwc.h [hunk 1]
```diff
@@ -34,6 +34,7 @@ class tehkanwc_state : public driver_device
 	void tehkanwc(machine_config &config);
 
 	void init_teedoff();
+	void init_teedoffj();
```
[the remark this answers is not in the document]
Comment (DavidHaywood) [reply src/mame/tecmo/tehkanwc.h]: yes, for some reason SourceTree was not picking up on the changed files, modifying them again fixed that.

**Pull request merged**